bugwarrior/command.py

```diff
@@ -28,6 +29,19 @@ def _get_section_name(flavor):
     return 'general'
 
 
+def _try_load_config(main_section, interactive):
+    try:
+        return load_config(main_section, interactive)
+    except IOError:
+        # Our standard logging configuration depends on the bugwarrior
+        # configuration file which just failed to load.
+        logging.basicConfig()
+
+        log.critical("Could not load configuration. "
```

Comment:
I attempted to address this in [irl#3](https://github.com/irl/bugwarrior/pull/3).